fastlane/lanes/release.rb

```diff
@@ -39,12 +39,22 @@
 
     new_version = release_version_current
 
+    # Delete all release notes metadata, including the source of truth.
+    # We'll generate a new source of truth next, and the localized versions will be re-downloaded once translated on GlotPress.
+    # It's important we delete them, otherwise we risk using old release notes for locales that won't get translated in time for the release finalization.
+    delete_all_metadata_release_notes
+
+    changelog_path = File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt')
     extract_release_notes_for_version(
       version: new_version,
-      release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt'),
-      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt')
+      release_notes_file_path: changelog_path,
+      extracted_notes_file_path: RELEASE_NOTES_SOURCE_PATH
+    )
+    # Add a new section to the changelog for the version _after_ the one we are code freezing
+    ios_update_release_notes(
```

Comment:
I had a quick look at the source for the only difference from its `android_update_release_notes` is the _internal_ logic they use to compute the version value to add at the top of the changelog file.

Given our move to the `Versioning` module, I think it would be appropriate to consolidate the two action into a single one, which receives either a precomputed next version (not to be confused with current `new_version` which is the value of the version we are code freezing and that the actions bump internally) or a `Versioning` object(s) to use for the computation.

I didn't go down that path in the interest of keeping momentum with the changes in this PR.